src/common/components/repositories-list/index.js

```diff
@@ -75,19 +66,19 @@ class RepositoriesList extends Component {
           </div>
         }
         <Table>
-          <THead>
-            <TR>
-              <TH>Name</TH>
-              <TH>Configured</TH>
-              <TH>Registered for publishing</TH>
-              <TH>Latest build</TH>
-            </TR>
-          </THead>
-          <TBody>
+          <Head>
+            <Row>
+              <Header col="30">Name</Header>
+              <Header col="20">Configured</Header>
+              <Header col="20">Registered for publishing</Header>
+              <Header col="30">Latest build</Header>
+            </Row>
+          </Head>
+          <Body>
```

Comment:
From code readability point of view I would prefer it to be `width="30%"` to be more explicit.
Only because these numbers add up to 100 I realised they are probably meant to be percent.

But it's not something I would block this PR on.